web/src/components/tasks/business-formation/BusinessFormationPaginator.test.tsx

```diff
@@ -121,6 +121,31 @@ describe("<BusinessFormationPaginator />", () => {
     initialUserData = generateUserData({ profileData, formationData });
   });
 
+  describe("search business name only", () => {
+    it("does not show buttons or stepper", () => {
+      preparePage(initialUserData, displayContent, undefined, undefined, undefined, undefined, true);
```

Comment:
I know this is pretty awful, but I also know that the in-progress Ethan work changes this function to named params, so I don't want to redo that work. It'll get changed as soon as that is merged